api/src/main/java/org/openmrs/module/eptsreports/reporting/calculation/generic/AgeOnMohMqArtStartDateCalculation.java

```diff
@@ -1,84 +0,0 @@
-package org.openmrs.module.eptsreports.reporting.calculation.generic;
-
-import java.util.Collection;
-import java.util.Date;
-import java.util.Map;
-import org.joda.time.Interval;
-import org.joda.time.Years;
-import org.openmrs.api.context.Context;
-import org.openmrs.calculation.patient.PatientCalculationContext;
-import org.openmrs.calculation.result.CalculationResult;
-import org.openmrs.calculation.result.CalculationResultMap;
-import org.openmrs.module.eptsreports.reporting.calculation.AbstractPatientCalculation;
-import org.openmrs.module.eptsreports.reporting.calculation.BooleanResult;
-import org.openmrs.module.eptsreports.reporting.calculation.melhoriaQualidade.MohMqInitiatedArtCalculation;
-import org.openmrs.module.eptsreports.reporting.utils.EptsCalculationUtils;
-import org.openmrs.module.reporting.common.Birthdate;
-import org.openmrs.module.reporting.data.person.definition.BirthdateDataDefinition;
-import org.springframework.stereotype.Component;
-
-@Component
-public class AgeOnMohMqArtStartDateCalculation extends AbstractPatientCalculation {
```

Comment:
@ningosi this is being used by quality improvements reports, did you see that?

Reply:
Yes, I saw that. I am the one who added that calculation that is no longer needed. I have cross referenced, build the code and tested.

Reply:
We changed that to use end of the reporting period, which we don't need  calculation to accomplish that, hence be removed